modules/peridynamics/doc/content/source/nodalkernels/PenaltyDirichletOldValuePD.md

```diff
@@ -0,0 +1,17 @@
+# Penalty Dirichlet Old Value Nodal Kernel
+
+!syntax description /NodalKernels/PenaltyDirichletOldValuePD
+
+## Description
+
+The `PenaltyDirichletOldValuePD` object is to apply Dirichlet boundary conditions of old variable solution using the penalty method for nodesets.
```

Comment:
This is the penalty method to impose Dirichlet BC but using nodal kernel for PD. To avoid rigid body motion for certain nodes, we have the RBMPresetOldValuePD class.